awswrangler/typing.py

```diff
@@ -165,7 +165,7 @@ class AthenaPartitionProjectionSettings(TypedDict):
     """
 
 
-class RayModinSettings(TypedDict):
+class RaySettings(TypedDict):
     """Typed dictionary defining the settings for distributing calls using Ray/Modin."""
```

Comment:
1. I think `RaySettings` makes more sense since I could see `parallelism` being useful for other functions as well. But I'm not sure, I think I've only seen it for `read_datasource`.
2. Yap I'll update the docstring, good catch